### DISCUSSION TITLE
Title components can use component prop with order to style as different types

### DISCUSSION THREAD

**Discussion by jacobsickels** [mantinedev/mantine#2084] [Feature requests]
### What package has an issue

@mantine/core

### Describe the bug

So, this may just be a typescript types issue, but I've found that you can use the Title with a component property and an order property to style the tags as different order types. 

For example this styles the h6 tag as an h1 tag.

`<Title component="h6" order={1}>{title}</Title>`

The Title component needs the `component` property in its types if we want to allow this functionality.

### In which browser did the problem occur

Chrome

### If possible, please include a link to a codesandbox with the reproduced problem

https://codesandbox.io/s/hungry-goldberg-hrk9be

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

Yes

**Comment by rtivital** [comment]
This feature will be added in one of the future 5.x releases
